tests/test_routes.py

```diff
@@ -67,12 +67,15 @@ def _create_wishlists(self, count):
     
     def _create_items(self, count):
         """ Factory method to create items in bulk """
+        wl = WishList()
```

Comment:
you need to create a wishlist with some row data: wl = WishList(name='wishlist1', category=category1')

Reply:
Actually, I think I resolved this error by adding a wishlist id. Now, I'm getting a foreign key error. 
<img width="1026" alt="Screen Shot 2021-03-31 at 3 30 12 PM" src="https://user-images.githubusercontent.com/25444717/113200244-14377480-9236-11eb-9f16-ca9494edbca9.png">